statsmodels/genmod/families/links.py

```diff
@@ -708,7 +740,24 @@ class probit(CDFLink):
 
     probit is an alias of CDFLink.
     """
-    pass
+
+    def inverse_deriv2(self, z):
+        """
+        Second derivative of the inverse link function
+
+        This is the derivative of the pdf in a CDFLink
+
+        """
+        return - z * self.dbn.pdf(z)
+
+    def deriv2_(self, p):
```

Comment:
why is there a trailing underscore?
was it left over from debugging?

Reply:
IIRC, because this is now inherited, but if we don't want to change CDFLink base class, then we can add this directly to probi.